Updated README.MD

MagicFab opened the pull request:
* Updated stars for Domoticz, HomeAssistant, OpenHab
* Added Mozilla WebThings gateway

MagicFab committed:
Updated README.MD

* Updated stars for Domoticz, HomeAssistant, OpenHab
* Added Mozilla WebThings gateway

pfalcon commented:
Thanks for the desire to help. However, it would be nice to make the changes more consistent (in particular, with existing changes):

1. Unrelated changes should go in separate commits.
2. Each commit should have clear description of what's being changed.
3. As long as some stars are being updated, it makes sense to update all.

MagicFab commented:
What are the unrelated changes ? I added an important project and updated the top ones. I didn't have more time to research all others. Can't remember the last time someone complained when getting a contribution, not matter how small.

pfalcon commented:
> I added an important project and updated the top ones.

Adding a project and update other projects are 2 unrelated changes, should go into separate commits.

> I didn't have more time to research all others.

There's no hurry, I myself wasn't able to update them in a while ;-).

> Can't remember the last time someone complained when getting a contribution, not matter how small.

Definitely not complaining, thanks again. But many projects have change control policy and contribution guidelines. Here's a random example for a project I contribute to: https://docs.zephyrproject.org/latest/contribute/index.html . And even small projects have such guidelines, and if people don't follow them, maintenance of a project becomes frustrating (literally, you spend your free time not on doing great things, like you dreamed, but on doing mediocre; a few people would agree to that, I guess).

pfalcon commented:
>  I myself wasn't able to update them in a while ;-).

All stars are now updated: https://github.com/pfalcon/awesome-smarthome/commit/e4825ce8770638e4792f4f020c6e01316be51710, thanks for driving that.

MagicFab commented:
Apologies for the way I snapped, reading back I wasn't expecting that. Thanks for completing the updates for other projects too!
